### DISCUSSION TITLE
Indicators on FakeReplayMode?

### DISCUSSION THREAD

**Discussion by LuanG01** [Mathieu2301/TradingView-API#100] [Q&A]
Hello,
is it possible to get indicator data in the FakeReplayMode to do backtesting with the free plan?
There are no indicators in the example and there is no proper doc for the library.

Thanks in advance

**Comment by Mathieu2301** [comment]
Hello, yes it's possible !

**Reply by LuanG01** [reply]
Can you or anyone else give me a short code snippet on how to do it? I couldnt figure it out yet
